node/src/chain/header_validation.rs

```diff
@@ -254,27 +254,30 @@ impl<'a, DB: database::DB> Validator<'a, DB> {
         Ok(merge_committees(&v_committee, &r_committee))
     }
 
-    /// Generates and returns voters
+    /// Extracts voters list of a block.
+    ///
+    /// Returns a list of voters with their credits for both ratification and
+    /// validation step
     pub async fn get_voters(
-        tip_block: &'a ledger::Header,
+        blk: &'a ledger::Header,
         provisioners: &Provisioners,
         prev_block_seed: Seed,
     ) -> anyhow::Result<Vec<VoterWithCredits>> {
-        let (_, _, v_committee, r_committee) = verify_block_att(
-            tip_block.prev_block_hash,
+        let (_, _, v_committee, r_committee) = verify_success_att(
+            blk.prev_block_hash,
             prev_block_seed,
             provisioners,
-            tip_block.height,
-            &tip_block.att,
-            tip_block.iteration,
+            blk.height,
+            &blk.att,
+            blk.iteration,
         )
         .await?;
 
         Ok(merge_committees(&v_committee, &r_committee))
     }
 }
 
-pub async fn verify_block_att(
+pub async fn verify_success_att(
```

Comment:
This function is used to verify failed iterations too, we should not rename it this way

Reply:
The rename is mistaken. It should have applied to `verify_winning_att`.